pcdsdevices/beam_stats.py

```diff
@@ -22,3 +22,14 @@ def __init__(self, prefix='', name='beam_stats', **kwargs):
     @property
     def hints(self):
         return {'fields': [self.mj.name]}
+
+
+class SxrGmd(Device):
+    mj = Cpt(EpicsSignalRO, 'SXR:GMD:BLD:milliJoulesPerPulse')
+
+    def __init__(self, prefix='', name='SxrGmd', **kwargs):
```

Comment:
I recommend following the `lowercase_underscore` name convention here instead of `CapitalizedCamelCase` because `ophyd` extends the names with underscores. So right now your inner signal's name is `SxrGmd_mj` which is a mix of the styles, rather than `sxr_gmd_mj`. This of course only matters if you're doing `bluesky` scans.